packages/browser-wallet/src/popup/pages/VerifiableCredential/VerifiableCredentialDetails.tsx

```diff
@@ -121,18 +182,31 @@ export default function VerifiableCredentialDetails({
         <>
             <Topbar
                 title={t('topbar.details')}
-                backButton={{ show: true, onClick: backButtonOnClick }}
+                backButton={{
+                    show: true,
+                    onClick: () => (showExtraDetails ? setShowExtraDetails(false) : backButtonOnClick()),
+                }}
                 menuButton={menuButton}
```

Comment:
I can still click on details, when I'm on the details page. :thinking: 
I think either that should be removed or changed to be a "see card front side" button. (If we change the button, maybe the back button should be changed to not treat details as a subpage)

Reply:
I don't like it but I'll have to go along with it